### DISCUSSION TITLE
Disable Request notification

### DISCUSSION THREAD

**Discussion by PixelAlarm** [photobooth-app/photobooth-app#285] [Q&A]
Hi,
sorry to bother, maybe I'm overseeing something :)

Is it possible to disable the Request notification?
Everytime an action is initialized it shows:
**Send successful
Request**

In my opinion this is redundant in some scenarios, since the countdown will indicate the success.
For collages it will even cover some of the UI-elements, if countdown is short.

![Screenshot from 2024-06-06 09-48-22](https://github.com/photobooth-app/photobooth-app/assets/158151736/c09d7d2f-4b7e-46d9-8051-8a8009bd71a2)

**Comment by Striker62** [comment]
hi in the admin section 
actions - collage 
 
Ask Approval Each Capture
Stop after every capture to ask user if he would like to continue or redo the capture. If disabled captures are granted as approved always.

**Comment by Striker62** [comment]
oh sorry you refering the green notice. ;-)

**Reply by PixelAlarm** [reply]
Yes, it's about the notice.
Thanks for replying though :)

**Comment by mgineer85** [comment]
Yes this we need to get rid of. Will check later today.

**Reply by mgineer85** [reply]
Is fixed now, release in processing... https://github.com/photobooth-app/photobooth-app/issues/286
